src/lib/input_schema.js

```diff
@@ -65,6 +71,36 @@ const readInputSchema = async (forcePath) => {
     };
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actorFolderDir
+ */
+const createPrefilledInputFileFromInputSchema = async (actorFolderDir) => {
+    const currentDir = process.cwd();
+    try {
+        process.chdir(actorFolderDir);
+        const { inputSchema } = await readInputSchema();
+        if (inputSchema) {
```

Comment:
If there is no input schema, we should create the `INPUT.json` anyway, with just `{}` in it, just like we do it now.

Reply:
Fixed and added additional tests for this

Reply:
Also added `--skip-dependecy-install` for `create` command test to make it slightly faster. Hopefully it is okay like this if not I will revert it